From dd638d9baac8eceb22d64d3ef90d46ddb17e61bc Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Wed, 2 Feb 2022 15:07:39 +0100
Subject: [PATCH 1/3] Add group audio to vendor.media.omx service

Resolves Qualcomm libs trying to access e.g. /dev/msm_alac and failing.

Referred from: https://source.codeaurora.org/quic/la/platform/frameworks/av/commit?id=150662d

Test: run vts -m VtsHalMediaOmxV1_0TargetAudioDecTest
Change-Id: I28dcc07c00f41ab542b24725d767227854f1b7c9
---
 services/mediacodec/android.hardware.media.omx@1.0-service.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/mediacodec/android.hardware.media.omx@1.0-service.rc b/services/mediacodec/android.hardware.media.omx@1.0-service.rc
index 845e5cc8f0..b5f9ab0cc7 100644
--- a/services/mediacodec/android.hardware.media.omx@1.0-service.rc
+++ b/services/mediacodec/android.hardware.media.omx@1.0-service.rc
@@ -1,6 +1,6 @@
 service vendor.media.omx /vendor/bin/hw/android.hardware.media.omx@1.0-service
     class main
     user mediacodec
-    group camera drmrpc mediadrm
+    group camera drmrpc mediadrm audio
     ioprio rt 4
     task_profiles ProcessCapacityHigh
-- 
2.30.2

